lib/model.d.ts

```diff
@@ -1839,15 +1842,19 @@ export abstract class Model {
    * Search for a single instance by its primary key. This applies LIMIT 1, so the listener will
    * always be called with a single instance.
    */
-  static findById<M extends Model>(this: {new (): M} & typeof Model, identifier?: number | string, options?: FindOptions): Promise<M | null>;
-  static findByPrimary<M extends Model>(this: {new (): M} & typeof Model, identifier?: number | string, options?: FindOptions): Promise<M | null>;
+  static findById<M extends Model>(this: {new (): M} & typeof Model, identifier?: number | string, options?: FindOptions & { rejectOnEmpty?: false }): Promise<M | null>;
+  static findById<M extends Model>(this: {new (): M} & typeof Model, identifier: number | string, options: FindOptions & { rejectOnEmpty: true }): Promise<M>;
```

Comment:
this forces to always set rejectOnEmpty explicitly, doesn't it? 🤔

Reply:
No, it's just on this overload. The overload above has it optional.